app/assets/stylesheets/modules/_landing.scss

```diff
@@ -0,0 +1,109 @@
+.hero {
+  background-color: $white;
+
+  &-title {
+    color: $dark-gray;
+    margin-bottom: em(5);
+    text-align: center;
+  }
+
+  &-subtitle {
+    font-style: italic;
+    font-weight: 300;
+    margin-top: em(5);
+    text-align: center;
+  }
+
+  &-main-text {
+    padding: 0 0 em(50);
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets